### PR TITLE
Adding Just Ape subecosystem & org

### PR DESCRIPTION
**Pull request by Just-Ape-Studios-Admin**
Adding Just Ape name & org

### CODE REVIEW

File: data/ecosystems/s/solana.toml [hunk 1]
```diff
@@ -121,6 +121,7 @@ sub_ecosystems = [
   "Jet Protocol",
   "Jito Network",
   "Jupiter Exchange",
+  "Just Ape","
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:124]: typo

Comment (eherrerosj) [reply data/ecosystems/s/solana.toml:124]: also, if you add an ecosystem here, there needs to exist a toml file with the same "title". I don't think we have a toml file for Just Ape. You have 2 options:
1. Delete this change and keep only the org change below
2. Create a toml file like just-ape.toml with the org in it and remove the org link from below

**Review by eherrerosj — CHANGES_REQUESTED** [review]
Left a comment

**Comment by jubos** [comment]
Going to close this out.  Please add the JustApe ecosystem repos and reopen when ready.

**Pull request closed without merging**